packages/ember-runtime/lib/is-equal.js

```diff
@@ -30,7 +33,7 @@
   ```
 
   @method isEqual
```

Comment:
Also needs an `@static` to show up as a function

Reply:
resolved